src/pages/Exhibition/index.tsx

```diff
@@ -1,20 +1,40 @@
 import React, { ReactElement, HTMLAttributes } from "react";
-import { Link } from "react-router-dom";
+import { useParams } from "react-router-dom";
 
-// import ExhibitionModel from "models/Exhibition";
+import ExhibitionLayout from "layouts/Exhibition";
+import { ExhibitionModel } from "models/Exhibition";
```

Comment:
With `export default`, this line should be changed to the following
```js
import ExhibitionModel from "models/Exhibition"; 
```

Reply:
Resolved in [aa7df74](https://github.com/niji-co/Fusion-Web/pull/74/commits/aa7df7486201cba8f957e788d72014da1a515d96)